### PR TITLE
Update calicoctl command reference for v3

### PR DESCRIPTION
**Pull request by gunjan5**
## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

File: master/reference/calicoctl/commands/get.md [hunk 1]
```diff
@@ -144,9 +139,9 @@ columns, and `custom-columns` for selecting which columns to display.
 Example
 ```
 $ calicoctl get hostEndpoint
-HOSTNAME   NAME        
-host1      endpoint1   
-myhost     myhost-eth0        
+NAME          NODE       
+endpoint1     host1         
+myhost-eth0   myhost
```
Comment (bcreane) [inline master/reference/calicoctl/commands/get.md]: spurious space eol

**Comment by emanic** [comment]
Awesome thanks for the PR @gunjan5

**Comment by emanic** [comment]
If possible, can you squash your commits? May be some broken links, as well.

**Comment by gunjan5** [comment]
@emanic just did :)

**Pull request merged**